build/Dockerfile

```diff
@@ -3,131 +3,171 @@
 
 # syntax=docker.io/docker/dockerfile:1.4
```

Comment:
```suggestion
# syntax=docker.io/docker/dockerfile:1
```

I suggest using only major version `1`, since it guarantees we'll have access to new functionalities and bugfixes from the builder.

https://docs.docker.com/build/dockerfile/frontend/#stable-channel